wgengine/wgtun/tun.go

```diff
@@ -0,0 +1,264 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package wgtun provides a TUN struct implementing the tun.Device interface
```

Comment:
Is it a `wgtun` or a `tstun`? 🤔

We already have a wireguard TUN package. It might be nice to have a more unique name.

/cc @danderson @crawshaw

Reply:
That actually sounds exactly right to me. Changed!